src/applications/personalization/profile-2/components/connected-apps/components/ConnectedApps.jsx

```diff
@@ -0,0 +1,130 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { ConnectedApp } from './ConnectedApp';
+import recordEvent from 'platform/monitoring/record-event';
+import { AppDeletedAlert } from './AppDeletedAlert';
+import {
+  deleteConnectedAccount,
+  dismissDeletedAccountAlert,
+  loadConnectedAccounts,
+} from 'applications/personalization/profile-2/components/connected-apps/actions';
+
+export class ConnectedApps extends Component {
+  componentDidMount() {
+    this.props.loadConnectedAccounts();
+  }
+
+  confirmDelete = accountId => {
+    this.props.deleteConnectedAccount(accountId);
+  };
+
+  dismissAlert = accountId => {
+    this.props.dismissDeletedAccountAlert(accountId);
+  };
+
+  render() {
+    const { accounts } = this.props;
+    const deletedAccounts = accounts
+      ? accounts.filter(account => account.deleted)
+      : [];
+    const activeAccounts = accounts
+      ? accounts.filter(account => !account.deleted)
+      : [];
+
+    return (
+      <div className="va-connected-acct">
+        <div className="">
+          {accounts && (
```

Comment:
we'll probably need a loading state for this component

Reply:
Yup, good point! Added 👍